### DISCUSSION TITLE
Custom header in Kong

### DISCUSSION THREAD

**Discussion by vietduc030496** [Kong/kong#12541] [Help] [answered]
Hello, i have a task configuration Kong:

Inspect WorkspaceB: A service is configured to proxy requests to http://mockbin.local:8000/anything The series can be accesses try doing a HTTP GET request to the following URL:							
	[http://localhost:8000/my-service]				
							
Expected Result							
1. When a request is sent to the service with a request header x-send-headers: true, the following headers must be sers by Kong:							
	• A Correlation Id header **_kong-request-id_** set to a uuid						
	• A Custom header **_x-custom-header_** set to hello-from-kong						
2. When a request is sent to the service without the **x-send-headers: true** request header present, neither the **kong-request- id** and **x-custom-header** should be sent.							
							
Instructions							
Observe the current behavior in the lab environment and make the necessary configuration changes to meet the expected result.

I don't have any idea for resolve task. Can you help me? thank you

**Comment by StarlightIbuki** [comment]
Do you have a specific problem/question with the task?

**Reply by vietduc030496** [reply, marked as answer]
i was resolver this problem with plugin
